.devcontainer/test-build.sh

```
@@ -0,0 +1,210 @@
+
+# Function to clean up
+clean_up() {
+    print_status "Cleaning up Docker resources..."
+    
+    cd "$(dirname "$0")"
+    
+    # Stop and remove containers
+    docker compose down --volumes --remove-orphans || true
+    
+    # Remove images
+    docker image rm $(docker images -q "*viscalyx*" "*devcontainer*") 2>/dev/null || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)